src/views/DataFile/DataFileTable.tsx

```diff
@@ -29,11 +29,15 @@ export function DataFileTable({ preview }: DataFileTableProps) {
           )}
         </Callout.Text>
       </Callout.Root>
-      <Table.Root size="1" variant="surface">
+      <Table.Root size="1" variant="surface" layout="auto">
         <Table.Header>
           <Table.Row>
             {preview.props.map((prop) => (
-              <Table.ColumnHeaderCell key={prop} uppercase={false}>
+              <Table.ColumnHeaderCell
+                key={prop}
+                uppercase={false}
+                maxWidth="300px"
```

Comment:
Added `maxWidth` to table columns to make sure it will look decent even with longer string values